pkgs/development/tools/build-managers/remake/default.nix

```diff
@@ -1,27 +1,68 @@
-{ lib, stdenv, fetchurl, readline }:
+{ lib
+, stdenv
+, fetchFromGitHub
+, fetchftpdir
+, autoreconfHook
+, gettext
+, perl
+, pkg-config
+, readline
+, texinfo
+}:
 
 stdenv.mkDerivation rec {
   pname = "remake";
-  remakeVersion = "4.1";
-  dbgVersion = "1.1";
+  remakeVersion = "4.3";
+  dbgVersion = "1.5";
   version = "${remakeVersion}+dbg-${dbgVersion}";
 
-  src = fetchurl {
-    url = "mirror://sourceforge/project/bashdb/remake/${version}/remake-${remakeVersion}+dbg${dbgVersion}.tar.bz2";
-    sha256 = "1zi16pl7sqn1aa8b7zqm9qnd9vjqyfywqm8s6iap4clf86l7kss2";
+  src = fetchFromGitHub {
+    owner = "rocky";
+    repo = "remake";
+    rev = "release_${remakeVersion}%2Bdbg-${dbgVersion}";
+    sha256 = "0fidz2jpy2c9pn16xv485jv575kikhs0dlcdh1qfl5l8xv1px79a";
+  };
+  po_files = fetchftpdir {
+    url = "http://translationproject.org/latest/make";
+    sha256 = "0y8sw9isjmbb8466lh14sk70zl49nyql2nf6494ln6083jzsgcil";
   };
 
   patches = [
     ./glibc-2.27-glob.patch
+    # remake tries to rsync the translations
+    # file from http://translationproject.org/latest/make
+    # to the po/ directory
+    # but the below postPatch does already that
+    ./no-rsync.patch
```

Comment:
:scream:

Reply:
Is this comment too wordy ?

Reply:
No, I was just screaming because upstream rsyncs stuff from the internet at build time (https://github.com/rocky/remake/issues/46). :smile: